manifests/enterprise/install.pp

```diff
@@ -0,0 +1,45 @@
+# Class splunk::enterprise::install
+#
+class splunk::enterprise::install {
+
+  if $facts['kernel'] == 'Linux' or $facts['kernel'] == 'SunOS' {
+    include splunk::enterprise::install::nix
+  }
+
+  $_package_source = $splunk::enterprise::manage_package_source ? {
+    true  => $splunk::enterprise::enterprise_package_src,
+    false => $splunk::enterprise::package_source
+  }
+
+  if $splunk::enterprise::package_provider != undef and $splunk::enterprise::package_provider != 'yum' and $splunk::enterprise::package_provider != 'apt' and $splunk::enterprise::package_provider != 'chocolatey' {
+    include archive::staging
```

Comment:
is this include actually required? :thinking:

Reply:
Good question, most of the crazy install logic I just forklifted and plopped into place...

Reply:
This module is listed as supporting Windows though.